pkg/endpoints/router.go

```diff
@@ -154,22 +155,34 @@ func (r Resource) RegisterExtension(container *restful.Container, namespace stri
 			extensions = append(extensions, ext)
 		}
 	}
-	logging.Log.Debugf("extension: %+v", extensions)
+	logging.Log.Debugf("Extension: %+v", extensions)
 	container.Add(ws)
+
+	logging.Log.Info("Adding API for extension")
+	wsv1 := new(restful.WebService)
+	wsv1.
+		Path("/v1/extension")
```

Comment:
minor point "/v1/extensions" might be nicer than  "/v1/extension" - also would need to change DEVELOPMENT.md to match and@AlanGreene would need to match endpoint in ui code.